pkg/converter/types.go

```diff
@@ -82,3 +102,58 @@ type UnpackOption struct {
 	// Timeout cancels execution once exceed the specified time.
 	Timeout *time.Duration
 }
+
+type TOCEntry struct {
+	/// Feature flags of entry
```

Comment:
Does comment style `///` has special use case? It's Rust style ....

Reply:
Fixed